node/heartbeat/sender.go

```diff
@@ -58,6 +59,7 @@ func (s *Sender) SendHeartbeat() error {
 
 	hb := &Heartbeat{
 		Payload: []byte(fmt.Sprintf("%v", time.Now())),
+		ShardID: (*s.shardCoordinator).SelfId(),
```

Comment:
no need of pointer if you do the suggested changes

Reply:
done